deploy/examples/create-external-cluster-resources.py

```diff
@@ -1257,15 +1273,17 @@ def validate_rgw_endpoint(self):
             return "-1"
 
         # check if the rgw endpoint is valid and belongs to the same cluster
-        fsid = self.get_fsid()
-        rgw_fsid, err = self.get_rgw_fsid(base_url, verify)
-        if err == "-1":
-            return "-1"
-        if fsid != rgw_fsid:
-            sys.stderr.write(
-                f"The provided rgw Endpoint, '{self._arg_parser.rgw_endpoint}', is invalid. We are validating by calling the adminops api through rgw-endpoint and validating the cluster_id '{rgw_fsid}' is equal to the ceph cluster fsid '{fsid}'"
-            )
-            return "-1"
+        # only check if info cap is supported
+        if info_cap_supported:
```

Comment:
What does it mean that it's supported? Is this set when the necessary rgw flags are set? Or when is it not supported?

Reply:
IMO this change is not related to multisite options, the rgw endpoint is validated by ceph cluster id, for fetching fsid from RGW server with help of `info` cap. This ability was added in the new version of ceph

Reply:
Right unrelated change